README.md

```diff
@@ -442,9 +441,22 @@ We can see that the `AccountIdFilter` requires an object that implements `IAccou
 
 **Note:** Only a single feature filter interface can be implemented by a single type. Trying to add a feature filter that implements more than a single feature filter interface will result in an `ArgumentException`.
 
+**Advanced:** Filters pf `IFeatureFilter` and `IContextualFeatureFilter` can share the same alias. Specifically, you can have one filter alias shared by 0 or 1 IFeatureFilter and 0 or N IContextualFeatureFilter<T>, so long as there is at most 1 applicable filter for T.

```

Comment:
Seems like it deserves it's own header.

Like ### Using contextual and non-contextual filters with the same name